libs/order/testing/src/factories/order-item.factory.ts

```diff
@@ -20,8 +20,8 @@ export class MockOrderItem implements DaffOrderItem {
   updated_at = faker.date.past();
   product_id = faker.random.number({min: 1, max: 1000});
   parent_item_id = faker.random.number({min: 1, max: 1000});
-  sku = 'sku';
-  name = 'Product Name';
+  sku = faker.random.alphanumeric(20);
```

Comment:
The appropriate function name here is `faker.random.alphaNumeric(20)` -- see https://github.com/marak/Faker.js/#api-methods

I'll open up an issue to discuss why we don't have typehinting here when we should.

Reply:
Maybe we should start a `@types/faker` contribution.

Reply:
fixed

Reply:
I believe one already exists and I presume either we don't use it, or somehow it's not loading correctly.

Reply:
bump